### DISCUSSION TITLE
Splitting `graphql_object` into multiple `impl` blocks for a single struct

### DISCUSSION THREAD

**Discussion by NeotenicPrimate** [graphql-rust/juniper#1079] [Q&A] [answered]
I am using macros to remove some of the boilerplate.
I'd like to derive some fields of a struct into one `graphql_object` impl block, but also be able to, in a separate `graphql_object` impl block, specify some additional computed fields.
Currently, if I generate the `graphql_object` impl block via macros, I cannot add any fields manually.

Is there any way to create multiple `graphql_object` impl blocks for a single struct?

I hope my question makes sense. Please tell me if you think I need to add information.

**Comment by ilslv** [comment, marked as answer]
Unfortunately we don't support multiple impl blocks for a single struct. See https://github.com/graphql-rust/juniper/issues/553 for more info.
